rapid7/metasploit-framework
# Exploit module for Delta Electronics Delta Industrial Automation COMMGR 1.08 Stack Buffer Overflow

Delta Electronics Delta Industrial Automation COMMGR 1.08 is affected by a stack-based buffer overflow vulnerability which can be leveraged by an attacker to execute arbitrary code. This module has been tested successfully on Windows XP SP3, Windows 7 SP1, and Windows 8.1. The vulnerable application is available for download at http://www.deltaww.com/Products/PluginWebUserControl/downloadCenterCounter.aspx?DID=7763&DocPath=1&hl=en-US.

## Verification

  1. Install Delta Industrial Automation COMMGR 1.08
  2. Start `msfconsole`
  3. Do ```use exploit/windows/scada/delta_ia_commgr_bof```
  4. Do ```set RHOST <target_ip>```
  5. Do ```run```
  6. You should get a shell. :)
